wallet/how-to/get-started-building/run-devnet.md

```diff
@@ -5,24 +5,40 @@ sidebar_position: 2
 
 # Run a development network
 
-You can run a personal Ethereum development network using [Ganache](https://www.trufflesuite.com/ganache),
+You can run a personal Ethereum development network using [Hardhat](https://hardhat.org/),
 which allows you to develop a dapp in a secure test environment.
 
 :::note
-When using a local development blockchain such as Ganache or
+When using a local development blockchain such as Hardhat or
 [anvil](https://book.getfoundry.sh/anvil/#overview-of-anvil), your node must calculate gas to make
 transactions on MetaMask.
 :::
 
-## Connect to Ganache
-
-Follow the [Ganache quickstart](https://trufflesuite.com/docs/ganache/quickstart/) to set
-up a development network.
+## Connect to Hardhat
+
+Follow the [Hardhat Quick Start](https://hardhat.org/hardhat-runner/docs/getting-started#quick-start)
+to set up a Hardhat development network.
+
+In your `hardhat.config.js` file, specify your
+[MetaMask seed phrase](https://support.metamask.io/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase)
+in the [`accounts`](https://hardhat.org/hardhat-network/docs/reference#accounts) field of the
+`networks.hardhat` configuration:
+
+```js title="hardhat.config.js"
+module.exports = {
+  networks: {
+    hardhat: {
+      accounts: {
+        mnemonic: "<your MetaMask seed phrase>",
+      },
+    },
+  },
+};
+```
 
-When you create a Ganache workspace, enter your MetaMask seed phrase into
-the **Account & Keys** setting.
-Ganache automatically gives each of your first 10 accounts 100 test ether (you can configure
-these numbers in **Accounts & Keys**), which makes it easy to start development.
+Hardhat automatically gives each of your first 20 accounts 10000 test ether (you can modify
+these numbers in the [`accounts`](https://hardhat.org/hardhat-network/docs/reference#accounts)
+configuration), which makes it easy to start development.
 
 :::caution important
 Your seed phrase controls all your accounts, so we recommend keeping at least one seed phrase for
```

Comment:
ah I see this note here now. I think this should be moved to the top of the section, as its very important that people read this before extracting their seedphrase.